test/helpers/cmd.go

```diff
@@ -26,52 +26,67 @@ import (
 	"k8s.io/client-go/util/jsonpath"
 )
 
-//CmdRes Command response with all the data
+// CmdRes contains the result from running a command.
 type CmdRes struct {
-	cmd    string
-	params []string
-	stdout *bytes.Buffer
-	stderr *bytes.Buffer
-	exit   bool
+	cmd    string        // Command to run
+	params []string      // Parameters to provide to command
+	stdout *bytes.Buffer // Stdout from running cmd
+	stderr *bytes.Buffer // Stderr from running cmd
+	exit   bool          // Whether command successfully ran.
```

Comment:
It's still not immediately clear which case is success and which is fail. This could be further improved by either renaming it `success`, or documenting that true is success (assuming that's actually the case).